packages/api/test/unit/beacon/oapiSpec.test.ts

```diff
@@ -86,6 +86,11 @@ const ignoredTopics = [
    topic block_gossip not implemented
    */
   "block_gossip",
+
```

Comment:
dev branch does not work as we are generally using the json file of the beacon spec which is only generated on release via swagger-cli. I have created a repo to host the json spec for now, this approach is already used in the open v2 apis branch, see https://github.com/ChainSafe/lodestar/pull/6951/files#diff-e95494735125652b2939787df6466e32cd940c1fbd0074200f616d87c6d6e6f6R26.

I have also asked in the E&R api channel if it's worth to cut a pre-release (similar to CL spec) but it does not seem like there is much interest as we are the only client that has api spec tests.

Reply:
Definitely in favor of a pre-release cycle that tracks the consensus-specs pre-release cycle

Maybe the reason we're the only client that has API spec tests is that the beacon API isn't very useful for testing since it rarely has releases that match the consensus-specs :joy:

Reply:
I wanted to look into beacon api tooling soon because right now it's really outdated, and keymanager api can't even be properly released right now since the release process is like 2 years old and not on par with how it's done on the beacon api spec.

I will try to push pre-releases as we definitely benefit from this, not only for new apis added for a hard fork but in general.

Reply:
We might have pre-releases soon (see https://github.com/ethereum/beacon-APIs/pull/463)